screen.go

```diff
@@ -90,7 +90,7 @@ func DisableMouse() Msg {
 	return disableMouseMsg{}
 }
 
-// disableMouseMsg is an internal message that that signals to stop listening
+// disableMouseMsg is an internal message that signals to stop listening
```

Comment:
Sometimes doubled `that` is intentional/necessary. This didn't feel like one of those cases.